--- sqldb.py	2024-08-18 16:40:12.000000000 +0200
+++ sqldb_copy.py	2024-12-03 17:37:50.559520200 +0100
@@ -203,7 +203,7 @@
 
         # opgelet deze regel is bewust een replace geen insert!
         self.insert_rec("replace into "+table+\
-            " values ( '128','" + const.P1_PATCH_LEVEL + "'                                     ,'Software patch:')" )
+            " values ( '128','" + const.P1_PATCH_LEVEL + "-e" + const.P1_FORK_LEVEL + "'        ,'Software patch:')" )
 
         self.insert_rec("insert or ignore into " + table + " values ( '129','0'                 ,'UI KWh meter productie(S0) zichtbaar 1/0)')")
 
